invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -385,6 +385,7 @@ class MetadataSchema(Schema):
     )
     version = SanitizedUnicode()
     rights = fields.List(fields.Nested(RightsSchema))
+    copyrights = SanitizedHTML(validate=validate.Length(min=3))
```

Comment:
"copyright" seems like a better label to me. My main argument is that since it's a single text field (not a list) the singular makes more sense. This is probably a bit of a US vs EU thing too (I've never used "copyrights" as a term). 

I'll also raise a minor point about the length restriction. Could it be one character in case someone just wanted to use the copyright symbol?

Reply:
done!